src/index.css

```diff
@@ -1,13 +1,150 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md) for this project, Please kindly ensure your  app is styled according to the [guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), checkout the screenshot below and kindly fix that. 

![ReactCapstone](https://user-images.githubusercontent.com/65852621/189445277-53d480b5-28f9-4714-88eb-22fce54c3b39.png)

Reply:
done